utils/utils.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 var WHITELIST = map[string]bool{
+	"stride1uk4ze0x4nvh4fk0xm4jdud58eqn4yxhrt52vv7": true,
```

Comment:
is this STRIDE_1 ? we should be careful about making sure this is removed before launching poolparty or mainnet, but both of those events are sufficiently rare where I don't think we need to change this

Reply:
Yeah it's local val1, we def need to remove it before mainnet, good flag